docker/dev/site.conf

```diff
@@ -79,7 +79,7 @@ server {
 
     # SUS-5792: rewrite for legacy corporate pages with short article path
     # see wgShortArticlePathWikis MediaWiki variable for the list of them
-    if ($host ~ ^(www|de|fr|pl|es|ja|ru|it).dev.wikia-local.com$) {
+    if ($host ~ "^(www|de|fr|pl|es|ja|ru|it)\.dev\.wikia-local\.com$") {
```

Comment:
This is broken. 
as this will break requests to e.g. api.php

I have just got to know that corporate pages are being migrated
https://wikia-inc.atlassian.net/browse/CT-2282

I propose that we do not handle them at all in the new setup. And treat this ticket as a blocker

Reply:
@hakubo - to some extent, we may treat CT-2282 as a blocker, depending on how long it's going to take. Apart from corporate pages, www.wikia.com has short article paths. Perhaps we should unify the configuration after the corporate pages are moved?